forge-core/src/main/java/forge/CardStorageReader.java

```diff
@@ -227,14 +227,11 @@ public final Iterable<CardRules> loadCards() {
         // Iterate through txt files or zip archive.
         // Report relevant numbers to progress monitor model.
 
-        final Set<CardRules> result = new TreeSet<>(new Comparator<CardRules>() {
-            @Override
-            public int compare(final CardRules o1, final CardRules o2) {
-                if (loadingTokens) {
-                    return String.CASE_INSENSITIVE_ORDER.compare(o1.getNormalizedName(), o2.getNormalizedName());
-                }
-                return String.CASE_INSENSITIVE_ORDER.compare(o1.getName(), o2.getName());
+        final Set<CardRules> result = new TreeSet<>((o1, o2) -> {
+            if (loadingTokens) {
+                return String.CASE_INSENSITIVE_ORDER.compare(o1.getNormalizedName(), o2.getNormalizedName());
             }
+            return String.CASE_INSENSITIVE_ORDER.compare(o1.getName(), o2.getName());
```

Comment:
What about `Comperator.comparing(CardRules::getName, String.CASE_INSENSITIVE_ORDER)` ?

Reply:
Looking at it, I think the `loadingTokens` check can also be moved out of the comparator, since that field is final.